packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -179,6 +179,65 @@ const Conversation = WebexPlugin.extend({
       });
   },
 
+  /**
+   * react to an activity
+   * @param {Object} conversation
+   * @param {Object} reactionPayload what we'll be sending to convo service to add or delete a reaction
```

Comment:
Let's use jsdoc's [@typedef](https://jsdoc.app/tags-typedef.html) and define a `ReactionPayload` type and use it here:

```suggestion
   * @param {ReactionPayload} reactionPayload what we'll be sending to convo service to add or delete a reaction
```
Something like
```js
/**
 * The payload to send a reaction
 * @typedef {Object} ReactionPayload
 * @property {Object} actor
 * @property {string} actor.objectType
 * etc
 */
```